granulate_utils/config_feeder/client/__init__.py

```diff
@@ -1,5 +1,13 @@
 from granulate_utils.config_feeder.client.bigdata import get_node_info
 from granulate_utils.config_feeder.client.client import ConfigFeederClient
 from granulate_utils.config_feeder.client.exceptions import APIError, ClientError, MaximumRetriesExceeded
+from granulate_utils.config_feeder.client.collector import ConfigFeederCollector
 
-__all__ = ["ConfigFeederClient", "ClientError", "APIError", "MaximumRetriesExceeded", "get_node_info"]
+__all__ = [
+    "ConfigFeederClient",
+    "ClientError",
+    "APIError",
+    "MaximumRetriesExceeded",
+    "ConfigFeederCollector",
+    "get_node_info",
```

Comment:
Should I expose `HttpClient` as well? for typing purposes (to be used by external collectors).

Reply:
maybe remove ConfigFeederCollector from here? these exports are for bare minimal usage. advanced users can import  directly from `granulate_utils.config_feeder.client.collector`